src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/runByPowerUntilCondition/ReverseRunFunnelUntilObjectOut.java

```diff
@@ -0,0 +1,11 @@
+package edu.greenblitz.robotName.commands.shooter.funnel.runByPowerUntilCondition;
+
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.Funnel;
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.FunnelConstants;
+
+public class ReverseRunFunnelUntilObjectOut extends RunFunnelByPowerUntilCondition {
+
+    public ReverseRunFunnelUntilObjectOut() {
+        super(FunnelConstants.EJECT_POWER, () -> !Funnel.getInstance().isObjectIn());
```

Comment:
Use “funnel” variable of from FunnelCommand

Reply:
You cant because `funnel` is null

Reply:
Why?

Reply:
because we do `funnel = Funnel.getInstance()` in the constructor of FunnelCommand so it is null before that

Reply:
That’s super funny 😂